blockchain/cfx.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/smartcontractkit/chainlink/core/logger"
 	"github.com/smartcontractkit/chainlink/core/store/models"
+	common2 "github.com/smartcontractkit/external-initiator/blockchain/common"
```

Comment:
I agree, and this will be changed once these other implementations are moved as well. I just quickly imported it like this so the whole project would compile for now.